src/App.tsx

```diff
@@ -1,18 +1,20 @@
 import './App.css'
-import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import { AuthProvider } from './context/AuthContext';
 import { LoginPage } from './pages/LoginPage';
 import { RegisterPage } from './pages/RegisterPage';
+import { ProtectedRoute } from './component/protectedRoute';
```

Comment:
Wrong import, should be `ProtectedRoute`, with capital letter, else wont compile.